shader/svgf_atrous.comp

```diff
@@ -1,185 +1,140 @@
 #version 460
 #extension GL_GOOGLE_include_directive : enable
+#extension GL_EXT_debug_printf : enable
```

Comment:
Does just enabling this extension have performance implications when it's not used? Maybe it should be disabled when we commit these.